doc/manual/packages.rst

```diff
@@ -570,11 +570,36 @@ going to have to do a *force push*:
   indicates that this should replace the ``fixbar`` branch found at
   ``myfork``.
 
-
 Creating a new Package
 ----------------------
 
-Guidelines for Naming a Package
+REQUIRE speaks for itself
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You should have a ``REQUIRE`` file in your package repo, with a bare minimum directive of what Julia
+version you expect your users to be running for the package to work. When Julia version ``0.x`` is released,
+use ``julia 0.x`` in your ``REQUIRE`` file. Typically, once a stable release is officially released,
+developers abaondon support for previous versions in future releases of their packages.
```

Comment:
Maybe just delete this last sentence since it's pretty well-covered by the next few paragraphs. We have a link to Travis below, but would be good to elaborate that it allows automatic testing against multiple Julia versions.